docs/api/range.md

```diff
@@ -34,6 +34,14 @@ import LabelsPlayground from '@site/static/usage/v7/range/labels/index.md';
 
 <LabelsPlayground />
 
+## Label Slot
```

Comment:
Threading for discussion:

I wonder if it makes more sense to have a "Labels" heading with Basic, Placement, and Custom HTML playgrounds.

We could also combine the Basic and Placement playgrounds into 1 playground that shows how to use the `label` property with placement options. If we went with the 2nd approach, it looks like we'd just need to change some headings on the page and keep the playgrounds as-is.

Reply:
As added context: My initial reaction was that it seems strange to have a Label Placement and Label Slot, but no "Label" section. Placement and Slot seem like they are sub-features for the overall "Label" feature.

Reply:
Rearranged things 👍